README.md

```diff
@@ -1,10 +1,17 @@
 # CLI_WordGuess
+
 A hangman style game played from the command line.
 
-Simply run npm install then node index.js and get playing!
+### Simply run:
+
+1. `npm install`
+2. `node index.js` or `npm start`
+3. and you're playing!
+
+## Technologies
+
+- [Node.js]()
+- [Inquirer]()
+- [Clear]()
 
-##Technologies
- Node.js
- inquirer.js
- 
-<a href="https://imgur.com/a/oCkPQLe">Screenshot of Gameplay</a>
+![Picture](./assets/wordGuessGame.png)
```

Comment:
This is the change, this can also be an html link too instead of the relative path:
ex:
`![Picture](https://imgur.com/a/oCkPQLe)`